tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
-envlist=py{34,35,36}-django{111,20},flake8
+envlist=py{35,36,37}-django{20,21,22}-wagtail{26,27}
 
 [testenv]
 commands=py.test --cov=wagtailbakery --cov-report=xml {posargs}
 deps=
-  django111: django>=1.11,<1.12
   django20: django>=2.0,<2.1
-  pytest
-  pytest-cov
-  pytest-django
+  django21: django>=2.1,<2.2
+  django22: django>=2.2,<2.3
+  wt26: wagtail>=2.6,<2.7
+  wt27: wagtail>=2.7rc1,<2.8
```

Comment:
This doesn't work and install 2.6 instead (see Travis build)

Reply:
Should be `wagtail26` and `wagtail27` I think, given the `envlist` line above?

Reply:
That's probably it 😅
Thanks for the help 🙇‍♂️